tests/models/glm5next/test_glm53_tp_pad.py

```
@@ -0,0 +1,59 @@
+# No-GPU unit test for the TP3 padding overlay: run inside the R17 image with the overlays mounted,
+#   VLLM_GLM53_TP_HEAD_PAD=72,66 TP=3, model dir at /model.
+import torch, vllm.distributed as dist
+dist.get_tensor_model_parallel_world_size = lambda: 3  # no process group in this test
+from vllm.transformers_utils.configs.glm5_next import Glm5NextConfig
+from vllm.transformers_utils.configs.glm53_tp_pad import pad_head_weights, routed_expert_intermediate, shared_expert_intermediate, vocab_padding_size
+c = Glm5NextConfig.from_pretrained("/model"); t = c.text_config
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)